build/i18n.m4

```diff
@@ -1,16 +1,14 @@
 AC_DEFUN([GP_I18N],
 [
-    if test -z "$conf_dir" ; then
-        conf_dir="."
-    fi
-    ALL_LINGUAS=`cd "$conf_dir/po" 2>/dev/null && ls *.po 2>/dev/null | $AWK 'BEGIN { FS="."; ORS=" " } { print $[1] }'`
     GETTEXT_PACKAGE=geany-plugins
     AC_SUBST(GETTEXT_PACKAGE)
     AC_DEFINE_UNQUOTED(
-        [GETTEXT_PACKAGE],
+    o    [GETTEXT_PACKAGE],
```

Comment:
Is this a typo or some special M4 syntax?

Reply:
typo, file will be removed